Documentation Lacking

Can you please document which packages are required to build xedgewarp?

Do we need development files for libX11, X11 Protocol headers or what?

```
$ make
rm -f xedgewarp obj/xedgewarp.o obj/pointer.o obj/extensions.o obj/randr.o obj/event.o obj/util.o obj/fake_outputs.o
rm -f man/*.1 man/*.xml
rm -f test/GLOB*
gcc -D'__VERSION="heads/master-0-g04ca182"' -Iinclude -std=gnu99 -Wall -Wundef -Wshadow -o obj/xedgewarp.o -c src/xedgewarp.c
In file included from include/all.h:4:0,
                 from src/xedgewarp.c:2:
include/externals.h:13:36: fatal error: X11/extensions/XInput2.h: No such file or directory
 #include <X11/extensions/XInput2.h>
                                    ^
compilation terminated.
Makefile:33: recipe for target 'obj/xedgewarp.o' failed
make: *** [obj/xedgewarp.o] Error 1

```

--- a/README.md
+++ b/README.md
@@ -36,12 +36,18 @@ xedgewarp is available in the AUR as [xedgewarp-git](https://aur.archlinux.org/p
 
 xedgewarp is make-based. Hence, you can clone the git repository and compile and install it via
 
-````
+```
 git clone https://github.com/Airblader/xedgewarp
 cd xedgewarp
 make
 sudo make install
 ```
+
+### Dependencies
+
+xedgewarp links with the following libraries:
+* [XCB](https://xcb.freedesktop.org/): libxcb-randr, libxcb-util, libxcb
+* [X11](https://xorg.freedesktop.org/): libX11-xcb, libX11, libXi
 
 ## Usage
 

